scripts/release/asana-create-tasks.js

```diff
@@ -39,52 +63,64 @@ const duplicateTemplateTask = (templateTaskGid) => {
 const run = async () => {
     setupAsana()
 
-    console.info('Asana on. Duplicating template task...')
-
+    // Duplicating template task...
```

Comment:
I've removed the logs because we're now reading the output of this script and I haven't found a better solution than to simply spit it out to the console. It's not like we were reading these logs anyway and if there's an error we print that and abort the action.